bounties/npm/@hapi/1/vulnerability.json

```diff
@@ -0,0 +1,52 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-05",
+    "AffectedVersionRange": "19.1.1",
+    "Summary": "Path traversal in @hapi/hapi via Symlink",
+    "Author": {
+        "Username": "vineetpandey",
+        "Name": "Vineet Pandey"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "@hapi/hapi",
+        "URL": "https://www.npmjs.com/package/@hapi/hapi"
+    },
+    "CWEs": [{
+        "ID": "CWE-200",
+        "Description": "Information Disclosure"
+    }],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "L",
+        "PR": "N",
+        "UI": "N",
+        "S": "U",
+        "C": "H",
+        "I": "N",
+        "A": "N",
+        "E": "F",
+        "RL": "",
+        "RC": "C",
+        "Score": "7.5"
+    },
+    "CVEs": [
+        ""
+    ],
+    "Repository": {
+        "URL": "https://hapijs.com/",
```

Comment:
Replace with URL to the GitHub repository

Reply:
Updated it.

Reply:
The latest commit does not show any updates.